scripts/light-godwoken-cli/package.json

```diff
@@ -1,30 +1,34 @@
 {
   "name": "light-godwoken-cli",
-  "version": "1.0.0",
+  "version": "1.1.0",
   "description": "Node scripts to make easy transactions via light-godwoken",
   "main": "dist/index.js",
   "scripts": {
     "cli": "ts-node-script src/index.ts",
-    "build": "npm run build:config-deps-devnet && npm run build:cli && npm link",
+    "build": "npm run build:cli && npm link",
     "build:cli": "esbuild src/index.ts --bundle --sourcemap --platform=node --outfile=dist/index.js --external:electron",
     "build:config-deps-devnet": "ts-node-script src/scripts/build-config-deps-devnet.ts"
   },
   "bin": {
-    "lgc": "bin/lgc"
+    "lgc": "bin/cli"
   },
   "devDependencies": {
     "@ckb-lumos/config-manager": "^0.18.0",
     "@ckb-lumos/lumos": "^0.18.0",
     "@ethersproject/abstract-signer": "^5.6.2",
     "esbuild": "^0.15.3",
-    "ts-node": "^10.4.0",
-    "typescript": "^4.5.4"
+    "keccak256": "^1.0.6",
+    "ts-node": "^10.9.1"
   },
   "dependencies": {
     "@ckb-lumos/base": "^0.18.0",
+    "@ckitjs/ckit": "^0.2.0",
+    "@polyjuice-provider/base": "^0.1.5",
```

Comment:
https://github.com/nervosnetwork/polyjuice-provider has been deprecated, please avoid using it if it is possible.

Reply:
Okay, I used this library to help me get `sudt_id` from a `sudt_type_args`, any better replacement?

Reply:
I think you could just add `getAccountIdByScriptHash` method into LightGodwokenProvider interface ?